### PR TITLE
Add withdraw example

### PR DESCRIPTION
**Pull request by amayvs**
**What this example does:**

* Withdraws from Paradex (Paraclear Contract) to L1 token bridge contract
* Waits for transaction to be accepted on L1 (Ethereum)
* Withdraw from L1 token bridge contract to L1 wallet

### CODE REVIEW

File: examples/withdraw.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    # Starknet account
+    account = get_account(
+        paradex_account_address, paradex_account_private_key_hex, paradex_config
+    )
+
+    amount = 1  # 1 USDC
+
+    # This method only waits for `ACCEPTED_ON_L2` status
+    # Status change to `ACCEPTED_ON_L1` could take up to 12 hours
+    client, withdraw_tx_hash = await withdraw_from_paraclear(
```
Comment (varunjain15) [inline examples/withdraw.py:131]: this implies that user already has funds on Paraclear
Suggestion - we either move it in try..catch or we first check whether Paraclear has withdrawable balance and exit if user doesn't have balance

File: examples/withdraw.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+    l1_bridge_address = config["bridged_tokens"][0]["l1_bridge_address"]
+    l1_bridge_abi = json.load(open(os.path.abspath("./abis/l1_bridge_abi.json")))
+    usdc_decimals = config["bridged_tokens"][0]["decimals"]
+
+    l1_contract = w3.eth.contract(address=l1_bridge_address, abi=l1_bridge_abi)
+
+    nonce = w3.eth.get_transaction_count(l1_recipient, "pending")
+    tx = {
+        "maxFeePerGas": w3.toWei("2", "gwei"),
```
Comment (varunjain15) [inline examples/withdraw.py:98]: fee amounts could be declared as class level constants...

**Review by varunjain15 — APPROVED** [review]
LG%C

File: examples/withdraw.py (hunk 1 above)
Comment (amayvs) [reply examples/withdraw.py:131]: Yes, this code sample assumes user has funds on Paraclear that they want to withdraw.

**Pull request merged**